pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeter/dependencies.yaml

```diff
@@ -15,16 +15,11 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Project
 metadata:
-  annotations:
-    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: project-${uniqueId}
 spec:
-  resourceID: ${TEST_DEPENDENT_FOLDER_PROJECT_ID}
```

Comment:
Q: Why are we changing these?

Reply:
It's brought up in this comment: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/1281/files#r1503617089 
It is unnecessary to use a pre-created project for this resource. I thought I had changed it in the above PR, but I may have forgotten to push my commit, or it was reverted after a rebase.